src/main/java/com/camilomelges/usersapi/UserController.java

```diff
@@ -0,0 +1,40 @@
+package com.camilomelges.usersapi;
+
+import lombok.Data;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@RestController(value = "users")
+@RequestMapping("users")
+public class UserController {
+
+  @GetMapping
+  public List<String> getAll() {
+      return List.of("1","2","3","4");
+  }
+
+  @RequestMapping(value = "{id}", method = RequestMethod.GET)
+    public String getById(@PathVariable("id") final String id) {
+        return id;
+    }
+
+    @RequestMapping(method = RequestMethod.POST)
+    public Map<String, String> post(@RequestBody final Map<String, String> user) {
+        user.put("id", RandomStringUtils.randomAlphabetic(1));
+        return user;
+    }
+
+    @RequestMapping(method = RequestMethod.PUT)
```

Comment:
Change it to a new implementation of Request mapping for get methods, @PutMapping

[READ](https://www.baeldung.com/spring-new-requestmapping-shortcuts)

Reply:
done